src/main/java/com/synopsys/integration/alert/web/api/role/util/PermissionModelUtil.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.web.api.role.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import com.synopsys.integration.alert.common.enumeration.AccessOperation;
+import com.synopsys.integration.alert.common.persistence.model.PermissionKey;
+import com.synopsys.integration.alert.common.persistence.model.PermissionMatrixModel;
+import com.synopsys.integration.alert.common.util.BitwiseUtil;
+import com.synopsys.integration.alert.web.api.role.PermissionModel;
+
+public class PermissionModelUtil {
```

Comment:
Didn't we already have something like this?

Reply:
convertToPermissionMatrixModel used to be a private method of the RoleActions. We wanted access to it in the test classes so we decided to move it into its own util class that can then be used by both rather than duplicating the code just for tests. If we already had a util class doing this type of work I may be unaware of it.